docs/user-manual/en/connection-ttl.md

```diff
@@ -186,4 +186,4 @@ thread adds a little more latency. These packets are:
 
 To disable asynchronous connection execution, set the parameter
 `async-connection-execution-enabled` in `broker.xml` to
-`false` (default value is `true`).
```

Comment:
what was changed on this line? I can't see it.

Reply:
NM: I see .. an enter was removed.. thanks

Reply:
That is weird. I'll remove it.